flickr.py

```diff
@@ -25,17 +25,18 @@ def params_unique_combination(baseurl, params_d, private_keys=["api_key"]):
             res.append("{}-{}".format(k, params_d[k]))
     return baseurl + "_".join(res)
 
-def search_flickr_by_tags(tags):
+def search_flickr_by_tags(tag, photo_id=None, method="flickr.photos.search", per_page=10):
```

Comment:
Are you sure you need per_page?

Reply:
I surely don't. Thanks so much for your impeccable insight